src/platform/datapath_winuser.c

```diff
@@ -4096,33 +4089,45 @@ CxPlatSocketSend(
 }
 
 void
-CxPlatDataPathSocketProcessSendCompletion(
+CxPlatDataPathSocketProcessQueuedSend(
     _In_ DATAPATH_IO_SQE* Sqe,
     _In_ CXPLAT_CQE* Cqe
     )
 {
+    UNREFERENCED_PARAMETER(Cqe);
     CXPLAT_SEND_DATA* SendData = CONTAINING_RECORD(Sqe, CXPLAT_SEND_DATA, Sqe);
     CXPLAT_SOCKET_PROC* SocketProc = SendData->SocketProc;
 
-#ifdef CXPLAT_DATAPATH_QUEUE_SENDS
```

Comment:
I forgot to in my PR, but could you remove `CXPLAT_DATAPATH_QUEUE_SENDS` from CMakeList.txt? That should be the last reference.

Reply:
Will do in my next PR - I don't want to restart the tests, and I have other PRs dependent on this one.